draft-ietf-quic-manageability.md

```diff
@@ -73,34 +73,32 @@ of QUIC-aware middleboxes, e.g. for load balancing.
 # Introduction
 
 QUIC {{?QUIC-TRANSPORT=I-D.ietf-quic-transport}} is a new transport protocol
-currently under development in the IETF QUIC working group, focusing on
-support of semantics as needed for HTTP/2 {{?QUIC-HTTP=I-D.ietf-quic-http}}.
-Based on current deployment practices, QUIC is encapsulated in UDP and
-encrypted by default. The current version of QUIC integrates TLS
+encapsulated in UDP and encrypted by default. QUIC integrates TLS
 {{?QUIC-TLS=I-D.ietf-quic-tls}} to encrypt all payload data and most control
-information.
+information. The design focused on support of semantics for HTTP, which required
+changes to HTTP known as HTTP/3 {{?QUIC-HTTP=I-D.ietf-quic-http}}.
 
 Given that QUIC is an end-to-end transport protocol, all information in the
 protocol header, even that which can be inspected, is not meant to be
 mutable by the network, and is therefore integrity-protected. While less
 information is visible to the network than for TCP, integrity protection can
-also simplify troubleshooting because none of the nodes on the network path
+also simplify troubleshooting, because none of the nodes on the network path
 can modify the transport layer information.
 
-This document provides guidance for network operation on the management of QUIC
+This document provides guidance for network operations that manage QUIC
 traffic. This includes guidance on how to interpret and utilize information that
-is exposed by QUIC to the network as well as explaining requirement and
-assumptions that the QUIC protocol design takes toward the expected network
-treatment. It also discusses how common network management practices will be
-impacted by QUIC.
+is exposed by QUIC to the network, requirements and assumptions that the QUIC
+design with respect to network treatment, and a description of how common
+network management practices will be impacted by QUIC.
 
 Since QUIC's wire image {{?WIRE-IMAGE=RFC8546}} is integrity protected and not
 modifiable on path, in-network operations are not possible without terminating
 the QUIC connection, for instance using a back-to-back proxy. Proxy operations
 are not in scope for this document. QUIC proxies must be fully-fledged QUIC
 endpoints, implementing the transport as defined in {{QUIC-TRANSPORT}} and
-{{QUIC-TLS}} as well as proxy-relevant semantics for the application(s) running
-over QUIC (e.g. HTTP/3 as defined in {{QUIC-HTTP}}).
+{{QUIC-TLS}}, proxy-relevant semantics for the application(s) running over QUIC
+(e.g. HTTP/3 as defined in {{QUIC-HTTP}}), and have the TLS credentials to
+authenticate as the server and (in some cases) client.
```

Comment:
Actually not sure here. The assumption wasn't necessary that the proxy can authenticate as the server (because then I guess it is the server) but probably rather that it would authenticate as an own entity offering a proxy service.

Reply:
"Transparent QUIC proxies"? You're right that if the client consents to a proxy (e.g. MASQUE) this isn't strictly true.

Reply:
I think the problem is about the term proxy. For me the term mean only that there is some entity between the client and the server. The whole point of this section was rather to say that a QUIC proxy cannot be transparent in a sense that it can terminate a connection without anybody noticing. And yes, I do have masque in mind.

Reply:
Ah now I see that you added the term "transparent QUIC proxy" in the text. Not sure if I want to coin that term here. Maybe we can just move the part about credentials into a separate sentence, e.g.:

```
A proxy can either explicit identify itself as providing a proxy service or may share the TLS credentials to
authenticate as the server and (in some cases) client acting as a front-facing instance for the offering service itself.
```